Rakefile

```diff
@@ -19,7 +23,7 @@ end
 task :submodules => ['_sass/brand/.git']
 
 task :dev => [:dependencies, :submodules] do
-  sh('bundle exec jekyll serve --drafts --config _config.yml')
+  sh('bundle exec jekyll serve --host 0.0.0.0 --drafts --config _config.yml')
```

Comment:
This means that the default local development server will be publicly visible from a developer's machine. I'm not sure what the security implications of that are, but likely not great given that `jekyll serve` is meant for development.

Was this considered when putting this PR together?

Reply:
It's a development server, but as it's mostly static, security issues should be low. Although I agree yes we might as well add in some protections where it's trivial.